src/components/card/index.tsx

```diff
@@ -5,7 +5,7 @@ interface Props {
   children?: React.ReactNode
 }
 
-const Container = styled.div`
+const Container = styled.ul`
```

Comment:
Care to explain this one for me?

Last I checked, the cards are being displayed as list items... So why are you turning it into an unordered list element?

Reply:
for the screen reader, when it's being read it will say the number/number.